packages/amplify-graphql-api-construct-tests/src/__tests__/backends/all-auth-modes/app.ts

```diff
@@ -16,8 +16,7 @@ const stack = new Stack(app, packageJson.name.replace(/_/g, '-'), {
 });
 
 const userPool = new UserPool(stack, 'TestPool', {});
-// const identityPoolId = new CfnIdentityPool(stack, 'TestIdentityPool', { allowUnauthenticatedIdentities: true }).attrId;
-const identityPoolId = 'us-west-2:54ee61c9-8d51-46aa-8898-33e8578d2f75'; // Random id, value not used in this test
+const identityPoolId = new CfnIdentityPool(stack, 'TestIdentityPool', { allowUnauthenticatedIdentities: true }).logicalId;
```

Comment:
Just making super sure this is supposed to be `logicalId` not `physicalId` or `attrId` of the construct? Logical IDs are allowed to be overridden, which I'm not expecting for the actual resolved Identity Pool ID. (Basically I was assuming that you'd be using the physical id)

Reply:
Ahh yeah, this should actually be `ref` which I tested locally, but haven't set here (just to make the test accurate).